docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -1334,6 +1334,19 @@ Query Parameters:
 
 -   **Content-type** – Set to `"application/tar"`.
 -   **X-Registry-Config** – base64-encoded ConfigFile object
+-   **X-Docker-BuildArgs** – base64-encoded JSON map of string pairs for build-time
```

Comment:
I think there was some discussion on this in the previous PR (not sure); is there a specific reason to pass these args as a *header*, and not just add it to the query-parameters? e.g. `&arg[foo]=bar&arg[bar]=baz`?